src/App.tsx

```diff
@@ -1,222 +1,137 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import classnames from 'classnames';
+import { createTodos, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+import { Header } from './components/Auth/Header';
+import { TodoList } from './components/Auth/TodoList';
+import { FooterTodo } from './components/Auth/FooterTodo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [value, setValue] = useState('');
+  const [islinkAll, setislinkAll] = useState(true);
+  const [islinkActive, setislinkActive] = useState(false);
+  const [islinkCompleted, setislinkCompleted] = useState(false);
+
+  const [currentFilter, setCurrentFilter] = useState(Filters.All);
+  const [isError, setisError] = useState(false);
+  const [isHidden, setisHidden] = useState(false);
+  const [completedTodo, setCompletedTodo] = useState<Todo>({
+    id: 0,
+    userId: 0,
+    title: '',
+    completed: false,
+  });
+
+  const handleCompletedTodo = () => {
+    setCompletedTodo((prevState) => ({
+      ...prevState,
+      completed: !prevState.completed,
+    }));
+  };
```

Comment:
I can toggle a completed status only once. And you didn't need to implement this feature in this part of the task. So fix it or remove it for now.

https://user-images.githubusercontent.com/99403198/210225384-526e8acc-a73c-46a0-810a-6da187b04831.mov